### PR TITLE
Fixes #34086 - explicitly depend on graphql-tag

### PR DESCRIPTION
**Pull request by evgeni**
We configure the graphql-tag loader in webpack, but didn't depend on it
in the past, which made builds using the loader (like REX plugin) fail.

Also adding `graphql` itself, as this is needed *by* the loader when
building stuff.

### CODE REVIEW

**Comment by evgeni** [comment]
For packaging, `depdendencies` vs `devDependencies` doesn't matter, but I'll leave it to @sharvit where which package belongs to.

**Comment by evgeni** [comment]
@sharvit what do you think? should we just get this in, and polish up things later?

**Review by ezr-ondrej — APPROVED** [review]
I guess both should be in dependencies, but I do not care much as it should be short-term workaround.

**Review by sharvit — APPROVED** [review]
Thanks @evgeni , works well 👍

**Comment by sharvit** [comment]
> @sharvit what do you think? should we just get this in, and polish up things later?

Yes, I can't merge the PR because it needs a packaging ack so I leave it to you.

**Comment by evgeni** [comment]
Argh, and I can't ack my own PR 🙈

@ekohl @ehelms plz?

**Review by pcreech — APPROVED** [review]

**Pull request merged**
